.github/workflows/OCV-Contrib-PR-3.4-macOS-ARM64.yaml

```diff
@@ -0,0 +1,366 @@
+    #   run: ./bin/opencv_perf_tracking --perf_impl=plain --perf_min_samples=1 --perf_force_samples=1 --perf_verify_sanity --skip_unstable=1 --gtest_filter=${{ env.GTEST_FILTER_STRING }}
+    #   working-directory: ${{ github.workspace }}/build
+    - name: Performance:video
+      if: ${{ always() && steps.build-opencv-contrib.outcome == 'success' }}
+      run: ./bin/opencv_perf_video --perf_impl=plain --perf_min_samples=1 --perf_force_samples=1 --perf_verify_sanity --skip_unstable=1 --gtest_filter=${{ env.GTEST_FILTER_STRING }}
+      working-directory: ${{ github.workspace }}/build
+    - name: Performance:videoio
+      if: ${{ always() && steps.build-opencv-contrib.outcome == 'success' }}
+      run: ./bin/opencv_perf_videoio --perf_impl=plain --perf_min_samples=1 --perf_force_samples=1 --perf_verify_sanity --skip_unstable=1 --gtest_filter=${{ env.GTEST_FILTER_STRING }}
+      working-directory: ${{ github.workspace }}/build
+    - name: Performance:xfeatures2d
+      if: ${{ always() && steps.build-opencv-contrib.outcome == 'success' }}
+      run: ./bin/opencv_perf_xfeatures2d --perf_impl=plain --perf_min_samples=1 --perf_force_samples=1 --perf_verify_sanity --skip_unstable=1 --gtest_filter=${{ env.GTEST_FILTER_STRING }}
+      working-directory: ${{ github.workspace }}/build
+    - name: Performance:ximgproc
+      if: ${{ always() && steps.build-opencv-contrib.outcome == 'success' }}
+      run: ./bin/opencv_perf_ximgproc --perf_impl=plain --perf_min_samples=1 --perf_force_samples=1 --perf_verify_sanity --skip_unstable=1 --gtest_filter=${{ env.GTEST_FILTER_STRING }}
+      working-directory: ${{ github.workspace }}/build
+    - name: Performance:xphoto
+      if: ${{ always() && steps.build-opencv-contrib.outcome == 'success' }}
+      run: ./bin/opencv_perf_xphoto --perf_impl=plain --perf_min_samples=1 --perf_force_samples=1 --perf_verify_sanity --skip_unstable=1 --gtest_filter=${{ env.GTEST_FILTER_STRING }}
+      working-directory: ${{ github.workspace }}/build
+    - name: Python3
+      if: ${{ always() && steps.build-opencv-contrib.outcome == 'success' }}
+      run: |
+        python3 ./test.py --repo ../../../ -v
+      working-directory: ${{ github.workspace }}/opencv/modules/python/test
+    - name: Java
+      id: java-test
+      if: ${{ always() && steps.build-opencv-contrib.outcome == 'success' }}
+      run: python3 ../opencv/modules/ts/misc/run.py . -a -t java
+      working-directory: ${{ github.workspace }}/build
+    - name: Save Unit Test Results
+      uses: actions/upload-artifact@v3
+      if: ${{ steps.java-test.outcome == 'success' && steps.build-opencv-contrib.outcome == 'success' }}
+      with:
+        name: junit-html
+        path: build/java_test/testResults/junit-noframes.html
+    - name: Pylint
+      if: ${{ always() && steps.build-opencv-contrib.outcome == 'success' }}
+      run: cmake --build . --config release --target check_pylint -- -j4
+      working-directory: ${{ github.workspace }}/build
```

Comment:
@alalek, do we need pylint test for macOS? I remember a discussion about a year ago, that we can keep `pylint` test only on Linux x86_64 and remove it from other pipelines.

Currently, we have `pylint` test on Linux ARM64, but only in OpenCV Contrib repository. I'd unify it to the one pattern - we have `pylint` test on each platform, or we run it only on Linux x86_64.

Reply:
Single platform / build configuration is enough.

Reply:
> Single platform / build configuration is enough.

Sure, `Pylint` tests are removed from all macOS workflows.